assets/controllers/member_modal_controller.js

```diff
@@ -0,0 +1,72 @@
+import { Controller } from '@hotwired/stimulus';
+
+/**
+ * @property {HTMLElement} modalTarget
+ * @property {HTMLElement} backdropTarget
+ * @property {String} memberValue
+ */
+export default class extends Controller {
+  static targets = ['modal', 'backdrop'];
+  static values = {
+    member: String,
+  };
+
+  connect() {
+    if (!this.hasModalTarget) {
+      throw new Error('Missing "modal" target.');
+    }
+    if (!this.hasBackdropTarget) {
+      throw new Error('Missing "backdrop" target.');
+    }
+
+    this.openers = Array.from(document.querySelectorAll('[data-open-member-modal]')).filter(
+      (el) => el.dataset.openMemberModal === this.memberValue,
+    );
```

Comment:
Can be simplified to:
```suggestion
    this.openers = Array.from(document.querySelectorAll(`[data-open-member-modal="${this.memberValue}"]`)
```